adam-core/src/main/scala/org/bdgenomics/adam/models/Interval.scala

```diff
@@ -41,4 +42,19 @@ trait Interval {
    */
   def width: Long = end - start
 
+  /**
+   * Need to implement getStart function from Locatable. (1-based start position. closed)
```

Comment:
Actually, think I may have misread.

> And is this the Interval that will need to go into bdg-utils to support IntervalRDD? Then there's no way I want to mix htsjdk stuff in.

@akmorrow13 @erictu, can you all comment?